### PR TITLE
Form Block: Allow Changing "Message Sent" Phrasing

### LINKED ISSUE #19640: Form Block: Allow customizing `Message Sent (go back)` that appears after form submission
#### Is your feature request related to a problem? Please describe.
There is currently no way to customize the title `Message Sent (go back)` that appears on the page after form submission. It would be nice to have this option, given that we can customize the message that appears below that title.

_Link: https://d.pr/i/3H6Sz9_

#### Describe the solution you'd like
Allow customizing `Message Sent (go back)` that appears after form submission. 

_Link: https://d.pr/i/UF9uQX_

#### Describe alternatives you've considered
None.

#### Additional context
This feature request was mentioned by a user in 25618708-hc

### PR DESCRIPTION
**Pull request by Aurorum**
Fixes #19640

#### Changes proposed in this Pull Request:

* When a user submits a form created with the Form Block, "Message Sent" will appear. The text below can already be customised, but this PR would allow users to change the phrasing in that heading.

#### Jetpack product discussion

N/A - this PR is in response to the original issue (and it's something I'd like to see too!)

#### Does this pull request change what data or activity we track or use?

No

#### Testing instructions:

Create a Form Block and verify that there's a new "Message Heading" option

Screenshot 2021-05-01 at 14 38 02

Enter some text there, and submit the form as a user. You should see that text appear once that form has been successfully submitted.

Screenshot 2021-05-01 at 14 39 25

### CODE REVIEW

File: projects/plugins/jetpack/modules/contact-form/grunion-contact-form.php [hunk 1]
```diff
@@ -2109,7 +2110,7 @@ static function parse( $attributes, $content ) {
 			$back_url = remove_query_arg( array( 'contact-form-id', 'contact-form-sent', '_wpnonce' ) );
 
 			$r_success_message =
-				'<h3>' . __( 'Message Sent', 'jetpack' ) .
+				'<h3>' . $form->get_attribute( 'customThankyouHeading' ) .
```
Comment (jeherve) [inline projects/plugins/jetpack/modules/contact-form/grunion-contact-form.php:2113]: Should we add some escaping here?

**Review by jeherve — CHANGES_REQUESTED** [review]
Thanks for the contribution!

You should now be able to edit `tools/eslint-excludelist.json` and remove `"projects/plugins/jetpack/extensions/blocks/contact-form/attributes.js",`, since there are no more errors in that file. 🥳

File: projects/plugins/jetpack/modules/contact-form/grunion-contact-form.php (hunk 1 above)
Comment (Aurorum) [reply projects/plugins/jetpack/modules/contact-form/grunion-contact-form.php]: Thank you! I've addressed both issues now.

**Review by kraftbj — APPROVED** [review]

**Comment by kraftbj** [comment]
We missed the tests. https://github.com/Automattic/jetpack/pull/19724/checks?check_run_id=2490641806

[This file](https://github.com/Automattic/jetpack/blob/7edde4bd462b8d3fda3290c8e72bd3e7b05e146d/projects/plugins/jetpack/extensions/blocks/contact-form/test/fixtures/jetpack__contact-form.json) and [this one](https://github.com/Automattic/jetpack/blob/7edde4bd462b8d3fda3290c8e72bd3e7b05e146d/projects/plugins/jetpack/extensions/blocks/contact-form/test/fixtures/jetpack__contact-form__deprecated-1.json) are failing since they don't include the new item.

**Comment by Aurorum** [comment]
Merged with `master` to ensure tests pass. :)

**Review by samiff — DISMISSED** [review]
@Aurorum this looks good to me! Can you merge master once more please.

**Comment by Aurorum** [comment]
Thank you - done! :)

**Comment by samiff** [comment]
Internal, WPcom Diff: D61201-code

**Comment by samiff** [comment]
@Aurorum Thank you for contributing! These changes should be part of the next 9.8 Jetpack release 👍

**Comment by jeherve** [comment]
r226303-wpcom

**Pull request merged**